### PR TITLE
Added an about page

### PR DESCRIPTION
**Pull request by tend2infinity**
Closes: #156

### Describe the changes you've made

I have moved some components to the about page and added some new components as well! I have used the GitHub API to get the information about the contributors. I have also added some styling libraries!

## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Enhancement in the existing features
- [ ] Other

## Mention any unusual behavior of your code (Write NA if not)
Any unusual behavior of your code

## Checklist:

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] My changes generate no new warnings.
- [x] I have verified/tested my code by running it locally.

## Video

https://user-images.githubusercontent.com/61948033/119116842-48652f80-ba46-11eb-8a02-672365ec356a.mp4

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -5,13 +5,13 @@ import './App.css';
 
 const Home = React.lazy(() => {
   return new Promise(resolve => {
-    setTimeout(() => resolve(import('./components/home')), 4000);
+    setTimeout(() => resolve(import('./components/home')), 2000);
```
Comment (Namanl2001) [inline src/App.js:8]: The timer's were set recently #197 please see this [comment](https://github.com/Namanl2001/MERN-Gurujii-dev/pull/197#discussion_r634404260)

File: src/App.js [hunk 2]
```diff
@@ -5,13 +5,13 @@ import './App.css';
...
   });
 });
 
 const Footer = React.lazy(() => {
   return new Promise(resolve => {
-    setTimeout(() => resolve(import('./components/Footer')), 5000);
+    setTimeout(() => resolve(import('./components/Footer')), 2000);
```
Comment (Namanl2001) [inline src/App.js:14]: duplicate

File: src/App.js [hunk 3]
```diff
@@ -21,13 +21,20 @@ const E404 = React.lazy(() => {
   });
 });
 
+const About = React.lazy(() => {
+  return new Promise(resolve => {
+    setTimeout(() => resolve(import('./components/about')), 1000);
```
Comment (Namanl2001) [inline src/App.js:26]: duplicate

File: src/App.js [hunk 4]
```diff
@@ -21,13 +21,20 @@ const E404 = React.lazy(() => {
...
+});
+
 function App() {
   return (
     <>
       <Router>
         <React.Suspense fallback={<Preloader />}>
           <Switch>
-            <Route exact path='/' component={Home} />
+            <Route exact path='/' component={About} />
```
Comment (Namanl2001) [inline src/App.js:36]: Looks like a good idea. Thanks

File: src/components/about.jsx [hunk 5]
```diff
@@ -0,0 +1,94 @@
+import React, { Component } from 'react';
+import Chatbot from '../components/chatbot';
+import WhatsAppIcon from '@material-ui/icons/WhatsApp';
+import FacebookIcon from '@material-ui/icons/Facebook';
+import LinkedInIcon from '@material-ui/icons/LinkedIn';
+import InstagramIcon from '@material-ui/icons/Instagram';
```
Comment (Namanl2001) [inline src/components/about.jsx:6]: We have recently moved this to seperate component see #203

**Review by Namanl2001 — CHANGES_REQUESTED** [review]
Great work! 🚀  @tend2infinity just some small nits else LGTM. Thanks

File: src/App.js (hunk 1 above)
Comment (tend2infinity) [reply src/App.js:8]: I was taking a lot more time to load the components that's why I changed, but nevermind I'll revert that

**Comment by tend2infinity** [comment]
Done the required changes

**Comment by sagar-barapatre** [comment]
resolve the conflicts.

**Comment by tend2infinity** [comment]
@Namanl2001 Done the required changes, sorry for being inactive for a while!

**Review by Namanl2001 — APPROVED** [review]
LGTM thanks.

**Pull request merged**